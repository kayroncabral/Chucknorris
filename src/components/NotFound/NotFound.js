import React from 'react'
import styles from './NotFound.scss'

const NotFound = () => (
  <div className={styles.NotFound}>
    <h1>Page Not Found :/</h1>
  </div>
)

export default NotFound
