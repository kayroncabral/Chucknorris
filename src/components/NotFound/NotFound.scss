@import '../../styles/styles.scss';

.NotFound {
  display: flex;
  justify-content: center;
  
  > h1 {
    color: white;
  }
}
