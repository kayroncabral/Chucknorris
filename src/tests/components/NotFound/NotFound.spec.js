import React from 'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import NotFound from '../../../components/NotFound'

test('should render Header correctly', () => {
  const renderer = new ReactShallowRenderer()

  renderer.render(<NotFound/>)
  expect(renderer.getRenderOutput()).toMatchSnapshot()
})
