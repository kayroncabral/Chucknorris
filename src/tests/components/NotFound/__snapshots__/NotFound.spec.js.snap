// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header correctly 1`] = `
<div
  className={undefined}
>
  <h1>
    Page Not Found :/
  </h1>
</div>
`;
